conf/config.level2.neon

```diff
@@ -28,6 +28,7 @@ rules:
 	- PHPStan\Rules\PhpDoc\IncompatiblePhpDocTypeRule
 	- PHPStan\Rules\PhpDoc\IncompatiblePropertyPhpDocTypeRule
 	- PHPStan\Rules\PhpDoc\InvalidPhpDocTagValueRule
+	- PHPStan\Rules\PhpDoc\InvalidPHPStanDocTagRule
```

Comment:
I'm guessing i need to add this to bleeding edge, but i'm not sure how that works, so i'd like some help with that 😅 @ondrejmirtes

Reply:
This is fine as a non bleeding edge thing. It’s not a mainstream syntax in PHP and people have no reason to have custom @phpstan- tags on purpose 😊